pkg/cloudprovider/suite_test.go

```diff
@@ -80,7 +80,9 @@ func TestAWS(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
-	env = coretest.NewEnvironment(coretest.WithCRDs(apis.CRDs...), coretest.WithCRDs(v1alpha1.CRDs...))
+	crds := test.RemoveNodeClassTagValidation(apis.CRDs)
+
+	env = coretest.NewEnvironment(coretest.WithCRDs(crds...), coretest.WithCRDs(v1alpha1.CRDs...))
```

Comment:
```suggestion
	env = coretest.NewEnvironment(coretest.WithCRDs(test.RemoveNodeClassTagValidation(apis.CRDs...)...), coretest.WithCRDs(v1alpha1.CRDs...))
```

nit: You could inline L83 into this one